commands/publish/index.js

```diff
@@ -126,7 +126,15 @@ class PublishCommand extends Command {
     });
 
     // cache to hold a one-time-password across publishes
-    this.otpCache = { otp: this.conf.get("otp") };
+    const otpRateLimitDelayCliParameter = this.options["otp-rate-limit-delay"];
```

Comment:
Yargs automatically camel-cases this to `otpRateLimitDelay` for us. Also, it's not _only_ provided by the CLI, as it can also be configured in lerna.json (`command.publish.otpRateLimitDelay`).